server/server.py

```diff
@@ -53,6 +54,9 @@
     csrf = SeaSurf(app)
     config = toml.load('config.toml')
     core = make_core(config)
+    slack_bot_channel = config['slack']['bot_channel']
```

Comment:
Can this be moved to `create_core()`?

Reply:
I agree - reading this should be in `make_core()` with all the other ones. If you do need the `bot` out here, it's better to access it via `core.bot` (or whatever you would need there to make that work).

Reply:
yes this can be done